### PR TITLE
feat: add reverse cache option

### PR DESCRIPTION
**Pull request by youen**

### CODE REVIEW

File: pkg/model/model_test.go [hunk 1]
```diff
@@ -572,3 +572,41 @@ func TestCacheShouldProvide(t *testing.T) {
 
 	assert.Equal(t, wanted, result)
 }
+
+func TestPipelineWithReverseProcessor(t *testing.T) {
```
Comment (youen) [inline pkg/model/model_test.go:576]: That test is for debug ?

File: internal/app/pimo/pimo.go [hunk 2]
```diff
@@ -29,28 +29,78 @@ import (
...
+				if !ok {
+					break
+				}
+				if pair.Key == "value" {
+					reverse.Set("key", pair.Value)
+				} else {
+					reverse.Set("value", pair.Value)
+				}
+			}
+			return reverse, nil
```
Comment (youen) [inline internal/app/pimo/pimo.go:57]:
```suggestion
			reverse := model.NewDictionary()
			reverse.Set("key", d.Get("value"))
			reverse.Set("value", d.Get("key"))
			return reverse, nil
```

File: internal/app/pimo/pimo.go [hunk 3]
```diff
@@ -29,28 +29,78 @@ import (
...
```
Comment (youen) [inline internal/app/pimo/pimo.go:96]:
```suggestion
			reverse := model.NewDictionary()
			reverse.Set("key", d.Get("value"))
			reverse.Set("value", d.Get("key"))
			return reverse, nil
```

**Comment by youen** [comment]
- [ ] Add entrie in CHANGELOG

File: pkg/model/model_test.go (hunk 1 above)
Comment (giraud10) [reply pkg/model/model_test.go:576]: yes it was to test, I remove it?

**Pull request merged**